payu/models/fms.py

```diff
@@ -92,8 +91,7 @@ def fms_collate(model):
                 mppnc_path = os.path.join(model.expt.lab.bin_path, f)
                 break
     else:
-        if not os.path.isabs(mppnc_path):
-            mppnc_path = os.path.join(model.expt.lab.bin_path, mppnc_path)
+        mppnc_path = model.expand_executable_path(mppnc_path)
```

Comment:
This is a random comment, but was chatting to @dougiesquire and @ezhilsabareesh8 today about their requirements for wave-watch three, and one of these was that there was an input file (a sort of derived config file) that must be generated every time the model is run, and it is a separate executable from the model executable, but should be built from the same source version as the model exe. So it was necessary to have this available in the `work` directory. So it would be a kind of generalisation of this, looking for tools in an executable path. 

I think it would have to be some new functionality, but basically just listing additional binary tools for a model and resolving their location and adding a link to it in `work`